public/locales/en.json

```diff
@@ -94,6 +94,8 @@
     "displayImage": "Display Image",
     "enterName": "Enter Name",
     "sort": "Sort",
+    "Latest": "Latest",
+    "Oldest": "Earliest",
```

Comment:
Change the key to"Earliest" to avoid confusion. In all the languages.

Reply:
Sure sir